python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -2,6 +2,7 @@
 
 import os
 import shutil
+import warnings
```

Comment:
usually, we use `from ..base import dgl_warning` in DGL.

Reply:
Got it.